src/views/sitemap.php

```diff
@@ -39,6 +40,37 @@
         <?php endif ?>
       </image:image>
     <?php endforeach ?>
+    <?php if ($__tag->isNews()): ?>
+    <?php 
+        /** @var $news \Watson\Sitemap\Tags\NewsTag */ 
```

Comment:
Don't need this here, just refer to `$__tag->getNews()->getName()` in the calls below.